domain-ee/ee-ep-merge-app/src/python_src/util/metric_logger.py

```diff
@@ -27,7 +27,12 @@
 
 ENV_TAG = f'environment:{ENV}'
```

Comment:
Should this be `env:` instead of `environment:`?
https://docs.datadoghq.com/tracing/guide/setting_primary_tags_to_scope/?tab=helm#environment

Reply:
Yeah, it probably should be!